src/gpujpeg_table.c

```diff
@@ -308,21 +308,30 @@ gpujpeg_table_huffman_encoder_init(struct gpujpeg_table_huffman_encoder* table,
 {
     assert(comp_type == GPUJPEG_COMPONENT_LUMINANCE || comp_type == GPUJPEG_COMPONENT_CHROMINANCE);
     assert(huff_type == GPUJPEG_HUFFMAN_DC || huff_type == GPUJPEG_HUFFMAN_AC);
+    assert(sizeof(table->bits) >= gpujpeg_table_huffman_y_dc_bits);
```

Comment:
there is missing sizeof around second argument

Reply:
That's what I get for adding asserts in the last minute ;) Fixed.